배포 시 contract 가 빌드되지 않아 배포가 실패하는 이슈

### 재현 방법

배포에서 start.sh 실행 도중 에러 발생합니다


### 발생 위치

scripts/start.sh
/Dockerfile

### 추가 정보

```
[stdout]#4 DONE 0.0s
[stdout]
[stdout]#5 [internal] load build context
[stdout]#5 transferring context: 13.46kB 0.0s done
[stdout]#5 DONE 0.0s
[stdout]
[stdout]#6 [pnpm-installed 2/5] RUN npm install --global pnpm
[stdout]#6 CACHED
[stdout]
[stdout]#7 [pnpm-installed 3/5] RUN pnpm config set store-dir .pnpm-store
[stdout]#7 CACHED
[stdout]
[stdout]#8 [prod 1/5] ADD . ./
[stdout]#8 CACHED
[stdout]
[stdout]#9 [pnpm-installed 5/5] WORKDIR /app
[stdout]#9 CACHED
[stdout]
[stdout]#10 [pnpm-installed 4/5] RUN pnpm install --global node-pre-gyp
[stdout]#10 CACHED
[stdout]
[stdout]#11 [workspace 1/2] COPY ./pnpm-lock.yaml .
[stdout]#11 CACHED
[stdout]
[stdout]#12 [workspace 2/2] RUN pnpm fetch --prod
[stdout]#12 CACHED
[stdout]
[stdout]#13 [prod 2/5] RUN pnpm -r install --frozen-lockfile --offline --prod
[stdout]#13 CACHED
[stdout]
[stdout]#14 [prod 3/5] RUN pnpm -r run build
[stdout]#14 1.383 Scope: 2 of 3 workspace projects
[stdout]#14 1.389 contracts build$ tsc
[stdout]#14 1.405 contracts build: sh: tsc: not found
[stdout]#14 1.408 undefined
[stdout]#14 1.409 /app/contracts:
[stdout]#14 1.409  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @jiphyeonjeon-42/contracts@0.0.11-alpha build: `tsc`
[stdout]#14 1.409 spawn ENOENT
[stdout]#14 ERROR: executor failed running [/bin/sh -c pnpm -r run build]: exit code: 1
[stderr]time="2023-09-06T08:42:51Z" level=warning msg="buildx: git was not found in the system. Current commit information was not captured by the build"
[stderr]failed to solve: executor failed running [/bin/sh -c pnpm -r run build]: exit code: 1
[stdout]------
[stdout] > [prod 3/5] RUN pnpm -r run build:
[stdout]#14 1.383 Scope: 2 of 3 workspace projects
[stdout]#14 1.389 contracts build$ tsc
[stdout]#14 1.405 contracts build: sh: tsc: not found
[stdout]#14 1.408 undefined
[stdout]#14 1.409 /app/contracts:
[stdout]#14 1.409  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @jiphyeonjeon-42/contracts@0.0.11-alpha build: `tsc`
[stdout]#14 1.409 spawn ENOENT
[stdout]------
```

--- a/package.json
+++ b/package.json
@@ -3,11 +3,11 @@
     "dev": "pnpm -r --parallel dev"
   },
   "dependencies": {
+    "typescript": "5.1.6",
     "zod": "^3.22.2",
     "@ts-rest/core": "^3.28.0"
   },
   "devDependencies": {
-    "typescript": "5.1.6",
     "@types/node": "18.16.1",
     "rome": "^12.1.3"
   }

